server/service/apple_mdm.go

```diff
@@ -551,6 +551,60 @@ func (svc *Service) GetMDMAppleProfilesSummary(ctx context.Context, teamID *uint
 	return ps, nil
 }
 
+type getMDMAppleFileVaultSummaryRequest struct {
+	TeamID *uint `query:"team_id,optional"`
+}
+
+type getMDMAppleFileVauleSummaryResponse struct {
+	Applied             uint  `json:"applied"`
+	ActionRequired      uint  `json:"action_required"`
+	Enforcing           uint  `json:"enforcing"`
+	Failed              uint  `json:"failed"`
+	RemovingEnforcement uint  `json:"removing_enforcement"`
+	Err                 error `json:"error,omitempty"`
+}
+
+// QUESTION: what is this line for?
+func (r getMDMAppleFileVauleSummaryResponse) error() error { return r.Err }
+
+// QUESTION: `errorer` naming? why this name?
+func getMdmAppleFileVaultSummaryEndpoint(ctx context.Context, request interface{}, svc fleet.Service) (errorer, error) {
+	req := request.(*getMDMAppleFileVaultSummaryRequest)
+
+	fvs, err := svc.GetMDMAppleFileVaultSummary(ctx, req.TeamID)
+	if err != nil {
+		return &getMDMAppleFileVauleSummaryResponse{Err: err}, nil
+	}
+
+	return &getMDMAppleFileVauleSummaryResponse{
+		Applied:             fvs.Applied,
+		ActionRequired:      fvs.ActionRequired,
+		Enforcing:           fvs.Enforcing,
+		Failed:              fvs.Failed,
+		RemovingEnforcement: fvs.RemovingEnforcement,
+	}, nil
```

Comment:
And here, instead of copying the values, you just do this:

```
return &getMDMAppleFileVauleSummaryResponse{
        MDMAppleFileVaultSummary: fvs,
}, nil
```